File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 31/07/2025, 21:17:29
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = len(nums)
        sm_nums = sum(nums)
        sm_real = sum(range(1, n+1))
        return sm_real - sm_nums

File: 268-MissingNumber/268-MissingNumber.java
// Last updated: 31/07/2025, 21:17:23
class Solution {
    public int missingNumber(int[] nums) {
        int sum1=0,sum2=0;
        for(int i=0;i<=nums.length;i++)
        {   
            sum1+=i;
        }
        for(int i=0;i<nums.length;i++)
        {
            sum2+=nums[i];
        }
        return sum1-sum2;
        
    }
}